nifi-commons/nifi-properties/pom.xml

```diff
@@ -21,4 +21,10 @@
         <version>1.0.0-SNAPSHOT</version>
     </parent>
     <artifactId>nifi-properties</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
```

Comment:
Should this be removed since we now have nifi-properties-loader in the framework which has all the bouncy castle stuff?